code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -455,7 +455,9 @@ GLOBAL_LIST_EMPTY(damage_icon_parts)
 	if(istype(glasses,/obj/item/clothing/glasses/mgoggles))
 		overlays_standing[GOGGLES_LAYER] = glasses.make_worn_icon(body_type = species.name, slot_name = slot_glasses_str, default_icon = 'icons/mob/eyes.dmi', default_layer = GOGGLES_LAYER)
 		apply_overlay(GOGGLES_LAYER)
-	
+	if(istype(glasses,/obj/item/clothing/glasses/hud/medgoggles))
```

Comment:
Turn this istype into a flag please

Reply:
I don't know what I am doing and could not figure out how to make a flag.